src/awesome_map/BoundaryInterceptor.js

```diff
@@ -252,6 +350,9 @@ define(function(require) {
 
             case EventTypes.MOUSE_WHEEL:
 
+                this._updateBoundariesStillVisible();
+                this._debouncedDetermineBoundaryVisibility();
```

Comment:
well one of the functions is debounced, so the order isn't particularly important, as update will always run immediately!  But i switched the order for consistency.  

determineBoundaryVisibility() creates a brand new list of the currently visible boundaries, but is debounced on each event fired so that events have to stop firing long enough in order for the list to update.  This eliminates the inertia bug, where one powerful scroll can cause events to fire from residual momentum upon reaching a boundary.

_updateBoundariesStillVisible() fires every time before event detection occurs to make sure that boundaries previously known to be visible are still visible, removing any that are not, but never adding new ones.  This is because it was possible to start at the top of the doc, scroll all the way to the bottom then scroll all the way to the top in one continuous event which would fire boundary events because the debounced boundary check never fired.  So i created a function which removes boundaries that have left visibility at any point.

It's a little wonky system, and i'm not sure i love it completely, but debouncing was the best solution @timmccall-wf and i could come up with for addressing the inertia problem.